library/ros

```diff
@@ -197,10 +197,10 @@ Directory: ros/crystal/ubuntu/bionic/ros-base
 
 Tags: dashing-ros-core, dashing-ros-core-bionic
 Architectures: amd64, arm64v8
-GitCommit: b832b00758b1cde2893b80f2f769b076164d9982
+GitCommit: 32244a8fafcf7d3c69c48e3a3aa082e4acc1fbf7
 Directory: ros/dashing/ubuntu/bionic/ros-core
 
 Tags: dashing-ros-base, dashing-ros-base-bionic, dashing
 Architectures: amd64, arm64v8
-GitCommit: b832b00758b1cde2893b80f2f769b076164d9982
+GitCommit: 32244a8fafcf7d3c69c48e3a3aa082e4acc1fbf7
```

Comment:
Shouldn't these commits be [`bf5aa6ca8ca41a096dd20b7b9e4e18594795bd27`](https://github.com/osrf/docker_images/commit/bf5aa6ca8ca41a096dd20b7b9e4e18594795bd27)?

Reply:
Indeed, something must have gone wrong in the library file generation, regenerated in https://github.com/docker-library/official-images/pull/6132/commits/d37d6b8190e5105705da26299a21eaa385a0a559
Thanks for catching that!